tsconfig.json

```diff
@@ -20,7 +20,28 @@
 		"noImplicitAny": false,
 		"noImplicitThis": false,
 		"esModuleInterop": true,
-		"outDir": "build"
+		"outDir": "build",
+		"baseUrl": ".",
+		"paths": {
+			"routes": [
+				"src/routes"
+			],
+			"middleware": [
+				"src/middleware"
+			],
+			"utils": [
+				"src/util"
+			],
+			"config": [
+				"src/config"
+			],
+			"interfaces": [
+				"src/interfaces"
+			],
+			"services": [
+				"src/services"
```

Comment:
Since we're redoing the folder structure, might as well add paths to make it easier to get to the imported files. 

Instead of doing `../../../../routes/api/api-router.ts` you can simply do `routes/api/api-router.ts`